kymatio/scattering1d/filter_bank.py

```diff
@@ -648,8 +584,20 @@ def scattering_filter_factory(J_support, J_scattering, Q, T, r_psi=math.sqrt(0.5
     https://tel.archives-ouvertes.fr/tel-01559667
     """
     # compute the spectral parameters of the filters
-    sigma_low, xi1, sigma1, j1s, xi2, sigma2, j2s = calibrate_scattering_filters(J_scattering, Q, T, alpha=alpha,
-                                                                                 r_psi=r_psi, sigma0=sigma0)
+    sigma_min = sigma0 / math.pow(2, J_scattering)
+
+    if isinstance(Q, tuple) and len(Q) == 1:
+        Q = Q + (1, )
+    elif isinstance(Q, int):
```

Comment:
Ideally, the Q check should only appear at:
https://github.com/kymatio/kymatio/blob/e0978a98a1b06768009344b246e7e64d0333e65b/kymatio/scattering1d/frontend/base_frontend.py#L27
when building the scattering1d object. But this may break the code of previous users when they create filter banks directly using `scattering_filter_factory`.  The test `test_numpy_scattering1d.py` indeed takes this case into account:

https://github.com/kymatio/kymatio/blob/e0978a98a1b06768009344b246e7e64d0333e65b/tests/scattering1d/test_numpy_scattering1d.py#L81-L100

This is why I add a Q check here besides the one in `base_frontend.py`.